tests/bdd/environment.py

```diff
@@ -13,8 +13,13 @@
 from importlib.machinery import SourceFileLoader
 
 from behave import *
-import psycopg2
-from psycopg2 import sql
+try:
+    import psycopg2 as psycopg
+    from psycopg2 import sql
+except ImportError:
+    import psycopg
+    from psycopg import sql
+
```

Comment:
I'd rather import 3 if both are available

Reply:
Order is the same as in osm2pgsql-replication. This is on purpose.